web/storagenode/src/storagenode/dashboard.ts

```diff
@@ -53,3 +54,21 @@ export class BandwidthInfo {
         this.remaining = available - used;
     }
 }
+
+/**
+ * AllowedVersion represents a minimal allowed version
+ */
+export class AllowedVersion {
+    public constructor(
+        public major: number,
+        public minor: number,
+        public patch: number) {}
+
+    /**
+     * Converts allowed version numbers to string type
+     * @returns allowed version - string of allowed version value
+     */
+    public toString(): string {
```

Comment:
normally those functions are just called String()

Reply:
but not in js, java or c#

Reply:
This function needs to change afaik with the use of the new struct